sr_port/mumps.hlp

```diff
@@ -6937,6 +6931,86 @@
    LOCK +^FAIL:1 which times out due to another process holding that lock. A
    ZSHOW "L" at this point displays the following output.
 
+   Example:
+
+   GTM>ZSHOW "I"
+   $DEVICE=""
+   $ECODE=""
+   $ESTACK=0
+   $ETRAP=""
+   $HOROLOG="64813,13850"
+   $IO="/dev/pts/0"
+   $JOB=20264
+   $KEY=""
+   $PRINCIPAL="/dev/pts/0"
+   $QUIT=0
+   $REFERENCE=""
+   $STACK=0
+   $STORAGE=2147483647
+   $SYSTEM="47,gtm_sysid"
+   $TEST=1
+   $TLEVEL=0
+   $TRESTART=0
+   $X=0
+   $Y=20
+   $ZA=0
+   $ZALLOCSTOR=671584
+   $ZB=""
+   $ZCHSET="M"
+   $ZCLOSE=0
+   $ZCMDLINE=""
+   $ZCOMPILE=""
+   $ZCSTATUS=0
+   $ZDATEFORM=0
+   $ZDIRECTORY="/path/to/the/current/directory"
+   $ZEDITOR=0
+   $ZEOF=0
+   $ZERROR="Unprocessed $ZERROR, see $ZSTATUS"
+   $ZGBLDIR="/path/to/the/global/directory/$gtmgbldir.gld"
+   $ZHOROLOG="64813,13850,790453,14400"
+   $ZININTERRUPT=0
+   $ZINTERRUPT="IF $ZJOBEXAM()"
+   $ZIO="/dev/pts/0"
+   $ZJOB=0
+   $ZKEY=""
+   $ZLEVEL=1
+   $ZMAXTPTIME=0
+   $ZMODE="INTERACTIVE"
+   $ZONLNRLBK=0
+   $ZPATNUMERIC="M"
+   $ZPIN="/dev/pts/0"
+   $ZPOSITION="+1^GTM$DMOD"
+   $ZPOUT="/dev/pts/0"
+   $ZPROMPT="GTM>"
+   $ZQUIT=0
+   $ZREALSTOR=694280
+   $ZRELDATE="20180614 00:33"
+   $ZROUTINES=". /usr/lib/fis-gtm/V6.3-005_x86_64 /usr/lib/fis-gtm/V6.3-005_x86_64/plugin/o(/usr/lib/fis-gtm/V6.3-005_x86_64/plugin/r)"
```

Comment:
Should $ZRELEASE be added in here along with any other SVNs we have added and changing the fis-gtm to something more yotta-ish?

Reply:
Note there are several fis-gtm references in the various help files.

Reply:
I meant all the fis-gtm references are now made yotta-ish.